README.md

```diff
@@ -130,11 +130,26 @@ enum Message {
 
 And it just works!
 
+You can also generate `From` and `TryFrom` traits for enum variants. Note that this will not work if enum has variants
+with the same field types. To use this feature add `impl_from_trait` attribute.
+```rust
+#[derive(ProtobufConvert)]
+#[protobuf_convert(source = "schema::Message"), impl_from_trait]
+enum Message {
+    Ping(Ping),
+    Pong(Pong),
+}
+```
+`From<Ping>`, `From<Pong>` and also `TryFrom<..>` traits will be generated.
+
+Another attribute that can be used with enum is `snake_case`. It instructs macros to generate snake case methods
```

Comment:
I think that we should you serde-like syntax https://serde.rs/field-attrs.html#rename to make code more obvious.